pennylane/tape/qscript.py

```diff
@@ -672,7 +672,7 @@ def set_parameters(self, params, trainable_only=True):
     # ========================================================
 
     @staticmethod
-    def _single_measurement_shape(measurement_process, device):
+    def _single_measurement_shape(measurement_process, batch_size, device):
```

Comment:
Though private, this method has an `Args` section in the docstring - could we have `batch_size` be described there?